server/src/main/java/org/opensearch/threadpool/ThreadPool.java

```diff
@@ -671,6 +672,9 @@ public void writeTo(StreamOutput out) throws IOException {
                     out.getVersion().before(LegacyESVersion.V_6_0_0_alpha1)) {
                 // 5.x doesn't know about the "fixed_auto_queue_size" thread pool type, just write fixed.
                 out.writeString(ThreadPoolType.FIXED.getType());
+            } else if (type == ThreadPoolType.RESIZABLE) {
+                // old vfi doesn't know about "resizable" thread pool. Convert RESIZABLE to FIXED
```

Comment:
Can you please use the full term instead of the acronym `vfi`? acronyms are sometimes a burden for developers to feel comfortable with the project, especially when they are in a comment.

Reply:
Thanks for pointing this out, I rephrased my comments and removed "vfi" from it